packages/@aws-cdk/aws-eks/test/integ.eks-cluster.ts

```diff
@@ -71,8 +68,6 @@ class EksClusterStack extends TestStack {
 
     this.assertServiceAccount();
 
-    this.assertServiceLoadBalancerAddress();
```

Comment:
I removed this assertion in favor of the new `integ.alb-controller.test` test. They basically test the exact same thing, but the `alb-controller` is not susceptible to https://github.com/aws/aws-cdk/issues/9970 because it uses a managed controller (instead of the unmanaged provider)